articles/Object-Detection-using-Fast-R-CNN-BrainScript.md

```diff
@@ -0,0 +1,537 @@
+---
+title:   Object  detection using Fast R-CNN
+author:    pkranen
+ms.author:   pkranen
+ms.date:   06/27/2017
+ms.custom:   cognitive-toolkit
+ms.topic:   conceptual
+ms.service:  Cognitive-services
+ms.devlang:   brainscript
+---
+
+# Object  detection using Fast R-CNN
+
+**Table of Contents**
+
+- [Summary](#summary)
+- [Setup](#setup)
+- [Run the toy example](#run-the-toy-example)
+- [Run Pascal VOC](#run-pascal-voc)
+- [Train CNTK Fast R-CNN on your own data](#train-on-your-own-data)
+- [Technical details](#technical-details)
+- [Algorithm details](#algorithm-details)
+
+## Summary
+
+<p align="center">
+<a target="_blank" href="./Tutorial_FastRCNN/svm_4WIN_20160803_12_37_07_Pro.jpg"><img src="./Tutorial_FastRCNN/svm_4WIN_20160803_12_37_07_Pro.jpg" alt="image" height="250"/></a>
+<a target="_blank" href="./Tutorial_FastRCNN/bus_01.jpg"><img src="./Tutorial_FastRCNN/bus_01.jpg" alt="image" height="250"/></a>
+</p>
+
+**This tutorial describes how to use CNTK Fast R-CNN with BrainScript and cntk.exe. Fast R-CNN using the CNTK Python API is described [here](./Object-Detection-using-Fast-R-CNN.md).**
+
+The above are examples images and object annotations for the grocery data set (left) and the Pascal VOC data set (right) used in this tutorial.
```

Comment:
>(left) and the Pascal VOC data set (right) [](start = 78, length = 42)

left and right are incorrect on smaller screens, then it is top or bottom.... 

perhaps (refrigerator image) and (street view) ???